src/Dashboard/Filter.php

```diff
@@ -58,235 +60,39 @@ class Filter extends \CommonDBChild
 
     /**
      * Return all available filters
-     * Plugins can hooks on this functions to add their own filters
      *
      * @return array of filters
      */
     public static function getAll(): array
     {
+        global $PLUGIN_HOOKS;
+        $more_filters = [];
+
         $filters = [
-            'dates'        => __("Creation date"),
-            'dates_mod'    => __("Last update"),
-            'itilcategory' => ITILCategory::getTypeName(Session::getPluralNumber()),
-            'requesttype'  => RequestType::getTypeName(Session::getPluralNumber()),
-            'location'     => Location::getTypeName(Session::getPluralNumber()),
-            'manufacturer' => Manufacturer::getTypeName(Session::getPluralNumber()),
-            'state'        => State::getTypeName(Session::getPluralNumber()),
-            'tickettype'   => _n("Ticket type", "Ticket types", Session::getPluralNumber()),
-            'group_tech'   => __("Technician group"),
-            'user_tech'    => __("Technician"),
+            DatesFilter::class,
+            DatesModFilter::class,
+            ItilCategoryFilter::class,
+            LocationFilter::class,
+            ManufacturerFilter::class,
+            RequestTypeFilter::class,
+            StateFilter::class,
+            TicketTypeFilter::class,
+            GroupTechFilter::class,
+            UserTechFilter::class,
         ];
 
-        $more_filters = Plugin::doHookFunction(Hooks::DASHBOARD_FILTERS);
-        if (is_array($more_filters)) {
-            $filters = array_merge($filters, $more_filters);
+        if (isset($PLUGIN_HOOKS[Hooks::DASHBOARD_FILTERS])) {
+            foreach ($PLUGIN_HOOKS[Hooks::DASHBOARD_FILTERS] as $hook_filters) {
+                foreach ($hook_filters as $filter) {
```

Comment:
Please verify here that corresponding plugin is active. Sometimes, plugins are registering their hook without being active.